StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Verification/LinkVerificationController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  LinkVerificationController.swift
+//  StripePaymentSheet
+//
+//  Created by Ramon Torres on 7/23/22.
+//  Copyright © 2022 Stripe, Inc. All rights reserved.
+//
+
+import UIKit
+
+/// Standalone verification controller.
+final class LinkVerificationController {
+
+    typealias CompletionBlock = (LinkVerificationViewController.VerificationResult) -> Void
+
+    private var completion: CompletionBlock?
+
+    private var selfRetainer: LinkVerificationController?
```

Comment:
nit: I'm not seeing where this is called, but, I'm hoping that this selfRetainer isn't needed.

Reply:
I think this is normal — we do this in our other modals to prevent them from being deallocated while on-screen.